OrbitGl/ImGuiOrbit.cpp

```diff
@@ -728,14 +738,51 @@ void OutputWindow::Draw(const char* title, bool* p_opened, ImVec2* a_Size) {
 
   ImGui::TextUnformatted(Buf.begin());
 
-  static bool checked = true;
-  ImGui::Checkbox("blah", &checked);
+  ImGui::End();
+  ImGui::PopStyleVar();
+}
+
+namespace ImGuiOrbit {
+std::string PrettyVariableName(const char* name) {
+  size_t length = strlen(name);
 
-  static int i1 = 0;
-  if (ImGui::SliderInt("slider int", &i1, 10, 100)) {
-    GCurrentTimeGraph->SetFontSize(i1);
+  // remove "m_"
```

Comment:
This feels a bit surprising. At least I would expect a comment what is going on here.

Reply:
yeah I removed the function altogether...

Reply:
Okay, thank!